src/deluge/model/instrument/midi_instrument.cpp

```diff
@@ -723,7 +723,7 @@ void MIDIInstrument::noteOffPostArp(int32_t noteCodePostArp, int32_t oldOutputMe
 
 	// Or, MPE
 	else {
-		if (oldOutputMemberChannel <= 14) {
```

Comment:
Have you checked if the change would be needed in lines 52 and 606?

Reply:
Yeah, and looks like there's a couple 1s that might need to be zero as well

Thanks for pointing it out!

Reply:
I tested it and it worked but there was just one small moment where one of the keys of the keyboard didn't trigger sound for a while. Maybe due to these extra changes needed, or maybe not. I will re-test again with your latest changes to see if I encounter any glitch in playing